Jenkinsfile

```diff
@@ -1,5 +1,5 @@
 def dockerHubRepo = "icgcargo/song-search"
-def githubRepo = "icgc-argo/song-search"
+def gitHubRepo = "icgc-argo/song-search"
```

Comment:
why is this cased like this?

Reply:
Not sure why... It looks like some of the other rdpc repos use this casing too in their Jenkins file. So maybe for consistency :man_shrugging: